app/Http/Transformers/ReportbackTransformer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Reportback;
+use League\Fractal\TransformerAbstract;
+
+class ReportbackTransformer extends TransformerAbstract
+{
+    /**
+     * Transform resource data.
+     *
+     * @param User $user
+     * @return array
+     */
+    public function transform(Reportback $reportback)
+    {
+        return [
+            'id' => (string) $reportback->id,
```

Comment:
Yeah, I'd say you only really need to cast if it's coming back as one thing and you need to set it as something else, but if it's the same type, maybe don't worry about it? :shrug: